device/celestica/x86_64-cel_seastone-r0/platform.json

```diff
@@ -3,194 +3,290 @@
         "Ethernet0": {
             "index": "1,1,1,1",
             "lanes": "65,66,67,68",
-            "alias_at_lanes": "Eth1/1, Eth1/2, Eth1/3, Eth1/4",
-            "breakout_modes": "1x100G[40G],2x50G,4x25G[10G]"
+            "breakout_modes": {
+                "1x100G[40G]": ["Eth1"],
+                "2x50G": ["Eth1/1", "Eth1/2"],
```

Comment:
Second interface should be Eth1/3

Reply:
One of our main vendor is actually use 1/1 and 1/2,  we probably will stick with that.  are you using this platform and have any concerns?

Reply:
We are keeping it as it is for Celestica platform.

Reply:
ok, so then 2 x 50g and 4x25g will have /2 starting at different lane?